pkg/minikube/registry/drvs/kvm2/kvm2.go

```diff
@@ -78,3 +85,33 @@ func createKVM2Host(mc config.MachineConfig) interface{} {
 		ConnectionURI:  mc.KVMQemuURI,
 	}
 }
+
+func status() registry.State {
```

Comment:
I would add a comment for status or choose a better name ?
does it check status of hyperkit or does it check if the binary is available? or does it check if it is out of date version?